### ISSUE TITLE
"Can't determine Firebase Database URL." in Functions emulator

### ISSUE THREAD

**Issue by samtstern** [firebase/firebase-tools#1261]
### [REQUIRED] Environment info

**firebase-tools:** 6.9.1

**Platform:** macOS

### [REQUIRED] Test case

**Functions**
```
const functions = require('firebase-functions');
const admin = require('firebase-admin');

admin.initializeApp();

exports.writeToDb = functions.https.onRequest(async (request, response) => {
  try {
    await admin.database().ref('write-test').set({
      date: new Date()
    });
    response.status(200).send('Wrote data!\n');
  } catch (e) {
    response.status(500).send(e);
  }
});
```

### [REQUIRED] Steps to reproduce

```
npx firebase emulators:exec "http http://localhost:5001/fir-dumpster/us-central1/writeToDb"
```

### [REQUIRED] Expected behavior

Write works

### [REQUIRED] Actual behavior

```
$ npx firebase emulators:exec "http http://localhost:5001/fir-dumpster/us-central1/writeToDb"
i  Starting emulators: ["functions"]
✔  functions: Using node@8 from host.
✔  functions: Emulator started at http://localhost:5001
i  functions: Watching "/private/var/folders/xl/6lkrzp7j07581mw8_4dlt3b000643s/T/tmp.wuBSDpsc/functions" for Cloud Functions...
⚠  Default "firebase-admin" instance created!
i  functions: HTTP trigger initialized at http://localhost:5001/fir-dumpster/us-central1/helloWorld
⚠  Ignoring trigger "firestoreFn" because the Cloud Firestore emulator is not running.
i  functions: HTTP trigger initialized at http://localhost:5001/fir-dumpster/us-central1/writeToDb
i  Running script: http http://localhost:5001/fir-dumpster/us-central1/writeToDb
i  functions: Beginning execution of "writeToDb"
⚠  Default "firebase-admin" instance created!
i  functions: Finished "writeToDb" in ~1s
{"code":"database/invalid-argument","message":"Can't determine Firebase Database URL."}✔  Script exited successfully (code 0)
i  Shutting down emulators.
i  Stopping functions emulator
```

**Comment by samtstern** [comment]
Originally reported by @Ihernandezpi in #1242

**Comment by abehaskins** [comment]
When initializing `firebase-admin` we [mistakenly drop user provided settings](https://github.com/firebase/firebase-tools/blob/master/src/emulator/functionsEmulatorRuntime.ts#L363) for un-mocked services. Once we merge these with the user provided settings the issue will be resolved.

**Comment by samtstern** [comment]
Fixed by #1263

**Comment by samtstern** [comment]
Version `6.9.2` has been released with this fix.
